NLog.Web.ASPNET5/LayoutRenderers/AspNetRequestValueLayoutRenderer.cs

```diff
@@ -137,4 +137,19 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             }
         }
     }
+
+    internal static class RequestAccessor
+    {
+        internal static HttpRequestBase TryGetRequest(this HttpContextBase context)
+        {
+            try
+            {
+                return context.Request;
+            }
+            catch (HttpException)
+            {
+                return null;
```

Comment:
(y) With or without any further explanation?

`InternalLogger.Debug("Exception thrown when accessing Request: " + ex);`
or 
` InternalLogger.Debug(ex.ToString());

Do you want a separate test verifying that it is logged? 

Current test passes based on the default log level for the internal logger discarding debug statements. A separate test can be added to verify that the internal logger do pick up on the exception if log level is set appropriately.